README.md

```diff
@@ -55,9 +55,109 @@ $ yarn audit
 ```shell
 $ yarn audit:fix
 ```
+### Setup AWS SSO Login
+```shell
+$ aws configure sso
+```
+- Choose session name: _{enter your name}_
+- Accept authorisation in browser
+- Choose the AWS account in the dropdown
+- CLI default client Region: eu-west-2
+- CLI default output format: json
+- aws sso login --profile _{profile name provided for account}
 
-## Additional Info
+## Architecture Diagram
+<img src="img.png" alt="img.png" width="500"/>
+
+## How To Deploy
+### 1. Setup Base CloudFormation Stacks
+To deploy the base common CloudFormation stacks required created by Dev Platform prior to deploying the solution use the Stack Orchestation tool provided in our [stack-orchestration](stack-orchestration) directory and deploy the [production_bootstrap.sh](stack-orchestration%2Fproduction_bootstrap.sh) script.
+
+***The stacks to be deployed are:***
+1. alerting-integration
+2. api-gateway-logs
+3. certificate-expiry
+4. vpc
+5. lambda-audit-hook
+6. checkov-hook
+7. infra-audit-hook
+
+#### Stack Orchestation Tool Prerequisites
+
+- [aws-vault](https://github.com/99designs/aws-vault)
+- [jq](https://formulae.brew.sh/formula/jq)
+
+#### ⚡ Prior to deploying check the latest version is being referenced which can be found in the [CHANGELOG](https://github.com/govuk-one-login/devplatform-deploy/tree/main)
+
+#### Steps:
+
+```shell
+$ cd stack-orchestation
+$ aws configure sso
+$ aws sso login --profile di-account-intervention-admin-324281879537
+$ sh production_bootstrap.sh
+```
+The bootstrap script would deploy all base Cloudformation stacks required for account set up.
+
+### 2. Deploy ais-infra stacks
+#### Clone the [ais-infra](https://github.com/govuk-one-login/ais-infra/blob/3046cb392b657ded2b652a52f0652a6ccd4d8630/) repo
+```shell
+git clone git@github.com:govuk-one-login/ais-infra.git
+cd ais-infra
+```
+##### ⚠️ Setup pre-commit
+```shell
+$ pre-commit install -f
```

Comment:
do you need pre-commit for this?

Reply:
That is very true! I think I just have secrets trauma 😅 And was thinking in case anyone clones the repo first time round, they would have pre-commit already set up just in case any changes will be made at any point